src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -71,3 +83,10 @@ const ProjectContainer = styled.div`
 const OptionGroupContainer = styled.div`
   padding: ${size.xs};
 `;
+
+const OverlineHover = styled.div`
```

Comment:
Could we apply these styled directly to `Overline` instead of a wrapping component. 
You can replace `styled.div` with `css` and pass the styled into `Overline`

Reply:
we only want the hover for the project settings page, we don't want it for the waterfall.

Reply:
It can still be a conditional style.

Reply:
I actually can't add it to Overline because we don't own it, it's part of leafygreen typography

Reply:
If you replace styled.div with `css` from `@emotion/core` and pass it into Overline as part of the `css` prop